### PR TITLE
1.0.0 RC

### PR DESCRIPTION
**Pull request by chrisbolin**
- readme
- package.json
- new handling for react elements (allowing deep equals testing without traversing circular structures

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -45,10 +45,12 @@ function equal(a, b) {
...
-      if (key === '_owner' && a[key]) {
-        // React-specific.
-        // avoid traversing circular reference for React elements with non-null _owner
-        if (a[key] !== b[key]) return false;
+      if (key === '_owner' && a.$$typeof && a._store) {
+        // React-specific: avoid traversing React elements' _owner.
+        //  _owner contains circular references
+        // and is not needed when comparing the actual elements (and not their owners)
+        // .$$typeof and ._store on just reasonable markers of a react element
+        // [no-op]
```
Comment (chrisbolin) [inline index.js]: hopefully you don't hate this design decision, but it's an empty block. it's just so much more readable to separate out all of the react stuff. we could also put a `continue` here.

**Comment by chrisbolin** [comment]
@ryan-roemer ready for your critique :)

**Comment by chrisbolin** [comment]
*NOTE* this also works on React@15. (I just tested :) )

File: README.md [hunk 2]
```diff
@@ -1,52 +1,83 @@
...
 
-To run benchmark (requires node.js 6+):
+To run benchmarks locally:
 
 ```bash
-npm install
-node benchmark
+yarn install
+yarn run benchmark
 ```
```
Comment (ryan-roemer) [inline README.md]: License at the bottom still needs to be changed. It's presently:

```markdown
[MIT](https://github.com/epoberezkin/fast-deep-equal/blob/master/LICENSE)
```

File: index.js (hunk 1 above)
Comment (ryan-roemer) [reply index.js]: Maybe `continue` _or_ something a little more verbose like:

```js
let isReactOwner;
// ....
// YOUR EXISTING COMMENT
isReactOwner = key === '_owner' && a.$$typeof && a._store;
if (!isReactOwner) {
  // all other properties should be traversed as usual
  if (!equal(a[key], b[key])) return false;
}
```

which gives you room for the future to add _other_ things to skip as well...

At any rate, you choice!

**Pull request merged**